LiteFarmOrg/LiteFarm
# fix: account info page user address char limit

To test:
- Go to my info page
- Enter user address longer than 255 characters
- You should get an inline error
